highway_env/road/graphics.py

```diff
@@ -125,14 +125,23 @@ def display(cls, lane: AbstractLane, surface: WorldSurface) -> None:
         stripes_count = int(2 * (surface.get_height() + surface.get_width()) / (cls.STRIPE_SPACING * surface.scaling))
         s_origin, _ = lane.local_coordinates(surface.origin)
         s0 = (int(s_origin) // cls.STRIPE_SPACING - stripes_count // 2) * cls.STRIPE_SPACING
+
+        last_pixel = [] # Store the pixel coordinates of the last pixel of a lane
         for side in range(2):
             if lane.line_types[side] == LineType.STRIPED:
                 cls.striped_line(lane, surface, stripes_count, s0, side)
             elif lane.line_types[side] == LineType.CONTINUOUS:
-                cls.continuous_curve(lane, surface, stripes_count, s0, side)
+                last_pixel.append(cls.continuous_curve(lane, surface, stripes_count, s0, side, return_last_pixel=True))
             elif lane.line_types[side] == LineType.CONTINUOUS_LINE:
                 cls.continuous_line(lane, surface, stripes_count, s0, side)
 
+        if lane.display_font_size != None:
+            x = (last_pixel[0][0] + last_pixel[1][0]) // 2 - 8
```

Comment:
I think that passing around this additional `return_last_pixel` argument and the associated returned value is a bit awkward, it adds unnecessary complexity, requires changing the several functions signature (which are now inconsistent) etc.
Instead, you can just recompute the coordinate here: `surface.vec2pix(lane.position(lane.length, 0))`